charts/castai-agent/templates/clustervpa-deployment.yaml

```diff
@@ -46,8 +46,10 @@ spec:
       affinity:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      {{- with .Values.securityContext }}
       securityContext:
-      {{- toYaml .Values.securityContext | nindent 8 }}
+        {{- toYaml . | nindent 8 }}
```

Comment:
what's the difference between `with .values.securityContext ....  toYaml .` and `toYaml .Values.securityContext` ?

Reply:
It won't even render the `securityContext` field if values are not given.